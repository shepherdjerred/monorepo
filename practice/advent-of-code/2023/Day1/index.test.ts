import { expect, test } from "bun:test";
import { solvePartOne, solvePartTwo } from ".";

test("part one example", async () => {
  expect(await solvePartOne("in.part1.example.txt")).toEqual(142);
});

test("part one", async () => {
  expect(await solvePartOne("in.part1.txt")).toMatchSnapshot();
});

test("part two example", async () => {
  expect(await solvePartTwo("in.part2.example.txt")).toEqual(281);
});

test("part two", async () => {
  expect(await solvePartTwo("in.part2.txt")).toMatchSnapshot();
});
