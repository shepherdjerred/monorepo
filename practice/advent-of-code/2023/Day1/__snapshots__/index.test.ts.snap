// Bun Snapshot v1, https://goo.gl/fbAQLP

exports[`part one 1`] = `54953`;

exports[`part two 1`] = `53868`;
